.buildkite/pipeline.yml

```diff
@@ -0,0 +1,23 @@
+steps:
+  - label: "generator"
+    plugins:
+      - JuliaCI/julia#v1:
+          persist_depot_dirs: packages,artifacts,compiled
+          version: "1.7"
+    agents:
+      queue: "juliaecosystem"
+      arch:  "x86_64"
+      os:    "linux"
+      # Only run on "sandbox.jl" machines
+      sandbox_capable: "true"
+    if: build.message !~ /\[skip tests\]/
+    timeout_in_minutes: 60
```

Comment:
Is this the timeout of the whole build?  We probably need to increase it a bit.

Reply:
No just the timeout for the generator itself, the other ones have time outs as well

Reply:
Oh, ok, at the moment we have it at 10 minutes: https://github.com/JuliaPackaging/Yggdrasil/blob/82b61df664f0cae541aa70486366cec0edbc90bc/azure-pipelines.yml#L247